pymatgen/ext/matproj.py

```diff
@@ -1081,7 +1081,7 @@ def get_cohesive_energy(self, material_id, per_atom=False):
         for el in comp_dict.keys():
             e = self._make_request("/element/%s/tasks/isolated_atom" % (el),
                                   mp_decode=False)[0]
-            isolated_atom_e_sum += e['output']["final_energy"] * comp_dict[el]
+            isolated_atom_e_sum += e['output']["final_energy_per_atom"] * comp_dict[el]
```

Comment:
Was this incorrect previously? Do we need to make a note of this anywhere?

Reply:
The cohesive energy uses the energy of an isolated atom as a reference, so most of the entries queried using _make_request will return an entry for such a system in which case it wouldn't matter if "final_energy" or "final_energy_per_atom" was used, however, queries for O, I, H, N, F returns two entries, one for isolated atom and another for isolated diatomic molecule which can lead to erroneous cohesive energy values. I'm currently discussing with @dwinston to get the latter removed from the API which should solve this problem.

Reply:
Understood, thank you for clarifying!